rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/rmw_common.hpp

```diff
@@ -1,313 +1,313 @@
-  const char * node_name,
-  const char * node_namespace,
-  rmw_names_and_types_t * service_names_and_types);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_get_subscriber_names_and_types_by_node(
-  const char * identifier,
-  const rmw_node_t * node,
-  rcutils_allocator_t * allocator,
-  const char * node_name,
-  const char * node_namespace,
-  bool no_demangle,
-  rmw_names_and_types_t * topic_names_and_types);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_service_server_is_available(
-  const char * identifier,
-  const rmw_node_t * node,
-  const rmw_client_t * client,
-  bool * is_available);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_destroy_subscription(
-  const char * identifier,
-  rmw_node_t * node,
-  rmw_subscription_t * subscription);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_subscription_count_matched_publishers(
-  const rmw_subscription_t * subscription,
-  size_t * publisher_count);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_take(
-  const char * identifier,
-  const rmw_subscription_t * subscription,
-  void * ros_message,
-  bool * taken);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_take_event(
-  const char * identifier,
-  const rmw_event_t * event_handle,
-  void * event_info,
-  bool * taken);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_take_with_info(
-  const char * identifier,
-  const rmw_subscription_t * subscription,
-  void * ros_message,
-  bool * taken,
-  rmw_message_info_t * message_info);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_take_serialized_message(
-  const char * identifier,
-  const rmw_subscription_t * subscription,
-  rmw_serialized_message_t * serialized_message,
-  bool * taken);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_take_serialized_message_with_info(
-  const char * identifier,
-  const rmw_subscription_t * subscription,
-  rmw_serialized_message_t * serialized_message,
-  bool * taken,
-  rmw_message_info_t * message_info);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_get_topic_names_and_types(
-  const char * identifier,
-  const rmw_node_t * node,
-  rcutils_allocator_t * allocator,
-  bool no_demangle,
-  rmw_names_and_types_t * topic_names_and_types);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_wait(
-  rmw_subscriptions_t * subscriptions,
-  rmw_guard_conditions_t * guard_conditions,
-  rmw_services_t * services,
-  rmw_clients_t * clients,
-  rmw_events_t * events,
-  rmw_wait_set_t * wait_set,
-  const rmw_time_t * wait_timeout);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_wait_set_t *
-__rmw_create_wait_set(const char * identifier, rmw_context_t * context, size_t max_conditions);
-
-RMW_FASTRTPS_SHARED_CPP_PUBLIC
-rmw_ret_t
-__rmw_destroy_wait_set(const char * identifier, rmw_wait_set_t * wait_set);
-
-}  // namespace rmw_fastrtps_shared_cpp
-
-#endif  // RMW_FASTRTPS_SHARED_CPP__RMW_COMMON_HPP_
+// Copyright 2016-2018 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Let's keep this change. This means that `rmw_common.hpp` was previously non-Unix line endings.